Client/src/Pages/Status/CreateStatus/index.jsx

```
@@ -0,0 +1,178 @@
+		});
+		if (newIdx !== -1) setTabIdx(newIdx);
+	};
+
+	const handleTabChange = (_, newTab) => {
+		setTabIdx(tabList.indexOf(newTab));
+	};
+
+	const handleSubmit = async (e) => {
+		e.preventDefault();
+		//validate rest of the form
+		delete form.logo;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)